### ISSUE TITLE
Issue of ParagraphVectors.inferVector(String newtext)

### ISSUE THREAD

**Issue by xinxu75** [deeplearning4j/deeplearning4j#4904]
#### Issue Description

I'm running DL4J on my personal laptop, within Eclipse IDE. When I tried to use ParagraphVectors.inferVector to infer a batch of new texts, I consistently had the following problems:

1) If I saved the ParagraphVectors model to a file and then loaded the model from the same file to call ParagraphVectors.inferVector, I received the error message of "A fatal error has been detected by the Java Runtime Environment". One error report is in attachment;
2) If I used the model immediately after building it, I didn't even receive any error message. The program simply stopped without error/warning at the point of ParagraphVectors.inferVector.

I noticed that this issue appears to be more likely to happen when the new text is a (slightly) longer sentence. The data for training the model and new texts are in Simplified Chinese, all being properly processed before using Dl4J.

If you could kindly provide a solution to this issue, or a guidance of workaround, that would be much appreciated

#### Version Information

* Deeplearning4j version: 0.8.0
* platform information (OS, etc):  DELL Inspiron 15 laptop with Windows 8 as OS.
* JVM argument: -Xmx16g

[hs_err_pid1480.log](https://github.com/deeplearning4j/deeplearning4j/files/1886879/hs_err_pid1480.log)

**Comment by saudet** [comment]
Please try again with Deeplearning4j 1.0.0-alpha.

**Comment by xinxu75** [comment]
I tried to upgrade to 1.0.0-alpha, however I found some big changes from v0.8 to v1, which causes my program no longer work

One example is org.deeplearning4j.text.sentenceiterator.labelaware.LabelAwareListSentenceIterator, which I cannot find in the new version, although your Javadoc still has this class?

Just wondering if you still have a similar class which allows iteration of each sentence (one sentence per line) with its corresponding label? Or maybe you don't have similar class as "ParagraphVectors" now no longer requires labels of each sentences, which is the way Doc2Vec algorithm was designed in the original paper?

If you could kindly provide some updated examples and clarity of how the data (sentences, labels, etc.) works now to train ParagraphVectors models in the 1.0.0-alpha version, that would be much appreciated.

**Comment by agibsonccc** [comment]
Yes this is natural. Major version bumps tend to do this in any software you use. Check our release notes: http://deeplearning4j.org/releasenotes and updated examples: https://github.com/deeplearning4j/dl4j-examples/tree/master/dl4j-examples/src/main/java/org/deeplearning4j/examples/nlp/paragraphvectors - we have more than enough for  you to migrate. Beyond that, I am closing this. There is nothing here concrete that should be considered a bug. If you have further questions, please ask in gitter.

**Comment by raver119** [comment]
Can you please post the code that reproduces this issue with 1.0.0-alpha version?

With best regards,
Vyacheslav K.

**Comment by xinxu75** [comment]
BTW, as suggested by Adams, I searched for release notes, which did not touch this class. I also checked the updated examples, they weren't affected because they never used this class. Given it's in v0.9.0 and there is no obvious reason to remove it, I'm suspicious that this is a mistake to exclude the LabelAwareListSentenceIterator class in 1.0.0-alpha? It may be worth looking into this? 

Thanks a lot for your help
